.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: go
-sudo: false
 go:
-  - '1.10'
+  - '1.13.x'
 
 before_install:
```

Comment:
I think that this stage is no longer necessary...

Reply:
yes not needed, dropped.

It could be used for caching but for a small module, it's faster to install fresh.